rustfmt.toml

```diff
@@ -7,7 +7,7 @@ imports_layout = "HorizontalVertical"
 merge_imports = true
 match_block_trailing_comma = true
 max_width = 120
-newline_style = "Unix"
+newline_style = "Native"
```

Comment:
Was `Unix` newlines causing an issue for you? I'd think it's better to have consistent newlines (i.e always `\n`) for all platforms. I think git will correct for this in either case.

Reply:
This is a problem for `cargo-fmt --all` as git automatically convert line endings to Windows-style when checking it out from GitHub. So running `cargo-fmt --all` changes all line endings in all files.

Reply:
Ok makes sense